src/utils/utils.py

```diff
@@ -199,7 +216,8 @@ def load_diagnostic_tsv(
         skip_rows=0,
         delimiter='\t',
         filter_idx=None,
-        filter_value=None):
+        filter_value=None,
+        tokenizer=''):
```

Comment:
See above - try to avoid setting defaults in multiple places, since it's easy for them to get out of sync. If it's just forwarded to process_sentence, could just forward additional kwargs.

Reply:
:+1: made it required